controllers/vcdmachine_controller.go

```diff
@@ -626,7 +626,7 @@ func CreateFullVAppName(ctx context.Context, cli client.Client, ovdcID string,
 	case "":
 		return vcdCluster.Name, nil
 
-	case infrav1beta3.DCGroup, infrav1beta3.ExternalLoadBalancer, infrav1beta3.UserSpecifiedEdgeGateway:
+	case infrav1beta3.DCGroup, infrav1beta3.ExternalLoadBalancer:
```

Comment:
We can remove `ExternalLoadBalancer` as well because that is a superset that nobody seems to be able to afford as of now. We could use a different ticket to do that.